### PR TITLE
[W5.2][W15-A2]Song Weiyang

### PR DESCRIPTION
**Pull request by sweiyang**

### CODE REVIEW

**Comment by devamanyu** [comment]
The format of the commit messages is slightly wrong. Please refer to this link for help : https://github.com/erlang/otp/wiki/writing-good-commit-messages.

**Review by devamanyu — COMMENTED** [review]
The SRP is well implemented. Please close this PR.

File: src/seedu/addressbook/ui/TextUi.java [hunk 1]
```diff
@@ -21,26 +21,10 @@
...
-
-    /** Offset required to convert between 1-indexing and 0-indexing.  */
-    public static final int DISPLAYED_INDEX_OFFSET = 1;
-
-    /** Format of a comment input line. Comment lines are silently consumed when reading user input. */
-    private static final String COMMENT_LINE_FORMAT_REGEX = "#.*";
 
     private final Scanner in;
     private final PrintStream out;
+    private final Formatter formatter = new Formatter();
```
Comment (devamanyu) [inline src/seedu/addressbook/ui/TextUi.java:27]: Its better practice to instantiate a Formatter object in the TextUI constructor.

**Comment by sweiyang** [comment]
Thank you

**Pull request closed without merging**